### PR TITLE
Add UNO-v8 Bridge

### PR DESCRIPTION
**Pull request by chase**
Merge after https://github.com/coparse-inc/libreofficekit/pull/10

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -283,6 +228,14 @@ jobs:
           if [ "${{ runner.os }}" = "Linux" ]; then
             cp -f ${{ inputs.working_dir }}/src/electron/{electron-api.json,electron.d.ts}{,.sha256sum} .
           fi
+      - name: Linux - Store LOK TypeScript bindings as artifact
+        if: ${{ runner.os == 'Linux' }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: lok-ts
```
[the remark this answers is not in the document]
Comment (chase) [reply .github/workflows/build.yml:235]: Out of scope for the PR

File: src/electron/office/document_client.cc [hunk 2]
```diff
@@ -771,4 +773,11 @@ bool DocumentClient::SendContentControlEvent(
 
   return true;
 }
+
+v8::Local<v8::Value> DocumentClient::As(const std::string& type, v8::Isolate* isolate)
+{
+  void *component = document_->getXComponent();
+  return convert::As(isolate, component, type);
```
Comment (whutchinson98) [inline src/electron/office/document_client.cc:780]: If you put in garbage here does electron do the error handling or does the program crash?

**Review by whutchinson98 — APPROVED** [review]
green light.

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -31,6 +31,13 @@ jobs:
       - name: Attach License
         run: |
           cp ./LICENSE ./src/electron/npm
+      - name: Download LOK TypeScript bindings artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: lok-ts
```
[the remark this answers is not in the document]
Comment (chase) [reply .github/workflows/publish.yml:37]: No, this is for reference purposes

File: src/electron/office/document_client.cc (hunk 2 above)
Comment (chase) [reply src/electron/office/document_client.cc:780]: There is no error, it returns undefined.

File: src/electron/office/document_client.h [hunk 4]
```diff
@@ -7,6 +7,7 @@
 
 #include <map>
 #include <string>
+#include <unordered_map>
```
[the remark this answers is not in the document]
Comment (chase) [reply src/electron/office/document_client.h:10]: Yes: https://github.com/coparse-inc/electron-libreoffice/blob/b0be9934bbce1c1121f358da5e9f3ccb85a5a71b/src/electron/office/document_client.h#L160

**Pull request merged**